starter-code/index.html

```diff
@@ -46,6 +46,13 @@
     <footer>
       &copy; 2016 Code Fellows | Static Blog | Happy Ipsum!
     </footer>
-    <!-- // TODO: Intentionally vague: include all necessary script tags, in proper order! -->
+    <script
+   src="https://code.jquery.com/jquery-3.1.1.min.js"
+   integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+   crossorigin="anonymous"></script>
+    <script src="scripts/blogArticles.js"></script>
+    <script src="scripts/article.js"></script>
+
```

Comment:
Great job on the dependencies! jQuery always needs to be first, or the jquery you write will not work :+1: